index.html

```diff
@@ -298,38 +298,38 @@ <h3 class="table-header">Saturday, October 30th</h3>
         <table class="table">
             <tr>
                 <td class="left">Midnight Snack</td>
-                <td class="middle">TBA</td>
+                <td class="middle">6th floor Commons</td>
                 <td class="right">12 AM</td>
             </tr>
 
             <tr>
                 <td class="left">Breakfast</td>
-                <td class="middle">TBA</td>
+                <td class="middle">6th floor Commons</td>
                 <td class="right">TBA</td>
             </tr>
 
             <tr>
                 <td class="left">Lunch</td>
-                <td class="middle">TBA</td>
+                <td class="middle">6th floor Commons</td>
                 <td class="right">TBA</td>
             </tr>
 
             <tr>
                 <td class="left">Sponsor Tech Talk</td>
                 <td class="middle">TBA</td>
-                <td class="right">TBA</td>
+                <td class="right">1:00 PM</td>
             </tr>
 
             <tr>
                 <td class="left">Sponsor Tech Talk</td>
                 <td class="middle">TBA</td>
-                <td class="right">TBA</td>
+                <td class="right">2:00 PM</td>
             </tr>
 
             <tr>
                 <td class="left">Sponsor Tech Talk</td>
                 <td class="middle">TBA</td>
-                <td class="right">TBA</td>
+                <td class="right">3:00 PM</td>
```

Comment:
What should we do about the other sponsor tech talks times? For Saturday should we add the other available times as well even tho not all may be filled out?